src/OneExplorer/ConfigObject.ts

```diff
@@ -210,41 +229,12 @@ export class ConfigObj {
    */
   private static parseBaseModels = (
     filePath: string,
-    iniObj: object
+    iniObj: object,
+    configSetting: ConfigSetting
   ): Artifact[] => {
```

Comment:
I attended that
- config object has config setting as member.
- when config object parse base models or products, it uses it's member config setting.

But parse function is static function, so it can't use member variable. So I add configSetting as input of parse function. Is there better way?

Reply:
I agree with the overal approach. I suggest one point: how about defining 'cfg type' and branch to call the corresponding configSetting in the function? To make our cfgobj instances lighter.